Description: Use SDL by default since it's faster without hardware acceleration
Author: Felix Geyer <fgeyer@debian.org>

---
 src/main.cpp |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

--- a/src/main.cpp
+++ b/src/main.cpp
@@ -241,8 +241,8 @@ void setupPHYSFS()
 		else
 		{
 			std::cerr << "Warning: Unknown renderer selected!";
-			std::cerr << "Falling back to OpenGL" << std::endl;
-			rmanager = RenderManager::createRenderManagerGL2D();
+			std::cerr << "Falling back to SDL" << std::endl;
+			rmanager = RenderManager::createRenderManagerSDL();
 		}
 	#else
 		#if MAC_OS_X
